app/src/main/java/fr/free/nrw/commons/utils/PermissionUtils.java

```diff
@@ -138,7 +138,8 @@ public void onPermissionRationaleShouldBeShown(PermissionRequest permission,
                         activity.getString(rationaleMessage),
                         activity.getString(android.R.string.ok),
                         activity.getString(android.R.string.cancel),
-                        token::continuePermissionRequest, token::cancelPermissionRequest);
+                        token::continuePermissionRequest, token::cancelPermissionRequest,
+                        null,false);
```

Comment:
Very minor: space after `,`
Or how about this?
```
token::continuePermissionRequest,
token::cancelPermissionRequest,
null,
false
```

Reply:
Yes, that looks better, I have made the changes!